test/test_models.py

```diff
@@ -280,6 +280,10 @@ def _check_input_backprop(model, inputs):
         "rpn_pre_nms_top_n_test": 1000,
         "rpn_post_nms_top_n_test": 1000,
     },
+    "vit_h_14": {
+        "image_size": 56,
+        "input_shape": (1, 3, 56, 56),
+    },
```

Comment:
@datumbox this according to your suggestions on changing the input image_size for the test on vit_h_14 models to speed up. The image_size need to be a multiple of patch_size which is 14, hence we use image_size of 56.

Reply:
Very interesting. Does this mean that the majority of the time is spent of the model initialization or on the JIT-script parsing?

Reply:
@datumbox I did a bit profiling locally and here are the results :
```
[2022-04-04 16:48:41.768663] Before building the model
[2022-04-04 16:48:45.000341] After building model
[2022-04-04 16:48:45.002153] After model.eval().to(device=dev)
[2022-04-04 16:48:45.207033] After doing model(x)
[2022-04-04 16:48:45.208375] After assert expected
[2022-04-04 16:48:45.208385] After assert shape num_classes
[2022-04-04 16:48:50.452526] After check_jit_scripttable
[2022-04-04 16:48:50.667378] After check_fx_compatible
[2022-04-04 16:48:51.256744] Finish
```
Seems like around 40% of the time is on building the model, and another 45% of the time is for `check_jit_scriptable`.